lib/chef/provider/package/yum.rb

```diff
@@ -29,8 +29,8 @@ class Package
       class Yum < Chef::Provider::Package
         include Chef::Mixin::GetSourceFromPackage
 
-        provides :package, platform_family: %w{rhel fedora amazon}
-        provides :yum_package, os: "linux"
+        provides :package, platform_family: %w{rhel fedora amazon aix}
+        provides :yum_package, os: %w{linux aix}
```

Comment:
this line should just be `provides :yum_package` there's actually little point in the os restriction and we've been removing those everywhere